additional locks needed in window

While `frame.Frame` is now thread-safe, other parts of Edwood are not. In particular, there is an easy-to-replicate problem:

* Start a child process that continuously generates output. I used `yes`.
* Move the `Window` to a different column.
* See crash:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x168 pc=0x117f805]

goroutine 50 [running]:
main.(*Window).SetTag(0xc420073800)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/wind.go:461 +0x75
main.xfidwrite.func1()
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:487 +0x20c
main.xfidwrite(0xc4202e8000)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:501 +0x438
main.xfidctl(0xc4202e8000, 0xc4200b80c0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:46 +0x87
created by main.xfidallocthread
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:628 +0x1e5
```

Conceivably, `Window` needs to be thread-safe?

--- a/xfid.go
+++ b/xfid.go
@@ -433,7 +433,9 @@ func xfidwrite(x *Xfid) {
 	//x.fcall.Data[x.fcall.Count] = 0; // null-terminate. unneeded
 	switch qid {
 	case Qcons:
+		global.row.lk.Lock()
 		w = errorwin(x.f.mntdir, 'X')
+		global.row.lk.Unlock()
 		updateText(&w.body)
 
 	case Qlabel:

